learn/safe-core/safe-core-account-abstraction-sdk/auth-kit.md

```diff
@@ -4,51 +4,104 @@ The [Auth kit](https://github.com/safe-global/safe-core-sdk/tree/main/packages/a
 
 Note: The Auth kit creates a [Signing Account, not a Smart Account](/learn/what-is-a-smart-contract-account.md#smart-accounts-vs-signing-accounts).
 
-The quick start guide below shows you how to sign transactions using your Signing Account. You can sign transactions using your Signing Account to create a Smart Account.
+The quick start guide below shows you how to create a Signing Account using your email or social media account. Once authenticated, you can sign transactions and interact with any Safe Smart Accounts you own.
 
 ## Quickstart
 
-### Prerequisites
+## Prerequisites
 
 - [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
-- [Create a Web3Auth account](https://web3auth.io) to use Web3Auth as the authentication provider.
+- [Create a Web3Auth account](https://web3auth.io)
+
+## Example
+
+See an [Auth Kit example in the Safe Core SDK](https://github.com/safe-global/safe-core-sdk/blob/7959821ab08c96cf3babb9ed906c01d644ac49f4/packages/auth-kit/example/src/App.tsx#L33) and [Pull Request 12](https://github.com/5afe/safe-space/pull/12) in [Safe Space](https://github.com/5afe/safe-space).
+
+
+{% embed url="https://user-images.githubusercontent.com/9806858/234088191-441c1c18-5922-47a4-a4dd-44fb383a54d6.mp4" %}
+[Watch video demo](https://user-images.githubusercontent.com/9806858/234088191-441c1c18-5922-47a4-a4dd-44fb383a54d6.mp4)
+{% endembed %}
```

Comment:
I think there is a misspell in this tag

Reply:
Which one is misspelled? `endembed`? That is a Gitbook code for embedding videos: https://github.com/safe-global/safe-docs/blob/main/learn/safe-core/safe-core-account-abstraction-sdk/onramp-kit.md?plain=1#L19